modules/document_repository/ajax/documentModifyCategory.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Document_repository module
+ *
+ * PHP Version 7
+ *
+ * @category Ajax
+ * @package  Loris
+ * @author   Loris Team <loris.info@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+$userSingleton =& User::singleton();
+if (!$userSingleton->hasPermission('document_repository_view')
+    && !$userSingleton->hasPermission('document_repository_delete')
```

Comment:
I think this should be `||` instead of `&&`

Reply:
I think you're right.